open-metadata-implementation/platform-services/platform-services-server/src/main/java/org/odpi/openmetadata/platformservices/server/OMAGServerPlatformOriginServices.java

```diff
@@ -8,7 +8,7 @@
  */
 public class OMAGServerPlatformOriginServices
 {
-    final String   implementationOrigin = "ODPi Egeria OMAG Server Platform (version 2.0-SNAPSHOT)\n";
+    final String   implementationOrigin = "ODPi Egeria OMAG Server Platform (version 2.0)\n";
```

Comment:
Don't like this hardcoding in the java file.
Maybe use in the future getClass().getPackage().getImplementationVersion(), of course we need to have the version in the package also something like maven-jar-plugin need to be used.

However, this is not a stopper for release.

Reply:
Yes I'd agree with that sentiment. 

We want to report the maven /project version here, and currently we don't add it in to the manifest.
```
 Manifest-Version: 1.0
 Created-By: Maven Jar Plugin 3.2.0
 Build-Jdk-Spec: 14
```
we'd need to add Implementation-Version

I opened up #940 ages ago to look at adding this/other info -- so let's continue the discussion there & if we can think how to get rid of it in more places the better --

Indeed not specific to this release